third_party/move/move-compiler/src/inlining/visitor.rs

```diff
@@ -533,26 +257,27 @@ impl<'l, V: TypedVisitor> TypedDispatcher<'l, V> {
     }
 
     pub fn sequence(&mut self, seq: &mut Sequence) {
-        let mut scoped = false;
+        let mut scope_cnt = 0;
```

Comment:
Is this for consistency with the other one at line 249?

Reply:
The difference shown is confusing, since the TypedDispatcher is gone.  The Dispatcher code had the scoped_cnt.  The TypedDispatcher had the simpler scoped predicate, which I assume probably doesn't work right in some cases of nested structures.  This diff is showing the diff between end of TypedDispatcher (old) with the end of Dispatcher (new).